src/panels/beaconSummaryPanel/BeaconSummaryPanel.test.tsx

```diff
@@ -0,0 +1,217 @@
+import { render, screen, waitFor } from "@testing-library/react";
```

Comment:
Large diff here because I've moved `BeaconSummaryPanel` 'stuff' into its own folder with its child editing state. 

![image](https://user-images.githubusercontent.com/54271433/117650023-1e7b5580-b188-11eb-9264-7179bff110b8.png)

New tests highlighted below.